frontend/src/admin/databases/components/DeleteDatabaseModal.jsx

```diff
@@ -52,6 +54,9 @@ export default class DeleteDatabaseModal extends Component {
                 closeFn={this.props.onClose}
             >
                 <div className="Form-inputs mb4">
+                    { database.name === "Sample Dataset" &&
```

Comment:
we should be checking for `database.is_sample` which is a more accurate way to tell if the database is the sample data.  the name is editable by the user.